src/Relation/Binary/Domain/Definitions.agda

```diff
@@ -0,0 +1,38 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Definitions for domain theory
+------------------------------------------------------------------------
+
+
+
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Relation.Binary.Domain.Definitions where
```

Comment:
Given that neither of the definitions here depend on `Poset` (now), maybe they should go where all the other binary relations are defined?